content/src/content/jcr_root/apps/core/wcm/components/download/v2/download/_cq_dialog/.content.xml

```diff
@@ -37,7 +37,7 @@
                                                 fieldLabel="Download Asset"
                                                 fileNameParameter="./fileName"
                                                 fileReferenceParameter="./fileReference"
-                                                mimeTypes="[image/*,video/*,audio/*,.pdf,.doc,.zip,.txt]"
+                                                mimeTypes="[image/*,video/*,audio/*,application/msword,application/vnd.openxmlformats-officedocument.wordprocessingml.document,.pdf,.zip,.txt]"
```

Comment:
I don't think that `.pdf,.zip,.txt` would work either as they are not proper mimeTypes.